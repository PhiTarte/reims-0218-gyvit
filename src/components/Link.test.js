//TODO TEST link

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import link from "./Link"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<link />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<link />).toJSON()
  expect(tree).toMatchSnapshot()
})
