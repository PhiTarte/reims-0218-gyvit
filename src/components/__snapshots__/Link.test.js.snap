// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NavbarLeftLink html 1`] = `<link />`;
